frontend/src/constants/StudentAssessmentConstants.ts

```diff
@@ -16,6 +16,14 @@ export const assessmentMetadata = {
   testName: "Unit 0 Review Test",
 };
 
+export const assessmentToolTips = {
+  questionTooltips: [
+    QuestionToolTips.MULTIPLE_CHOICE,
+    QuestionToolTips.MULTI_SELECT,
+    QuestionToolTips.SHORT_ANSWER,
+  ],
+};
+
 export const typeToImageMetadata = {
```

Comment:
you can just update this to `responseTypeMetadata` and add a `tooltip` field, i.e `tooltip: "Users will have to select the best response from a list of options"` under **Multiple Choice**.

then, you can delete `asssessmentToolTip` in constants, `QuestionToolTips` in question types and don't really need to update  the parameters in `StudentAssessment` and `QuestionTypeImages`

Reply:
ok!